bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -215,7 +228,16 @@ public boolean equals(Object obj) {
     @Override
     public String toString() {
         return "DiscoveryResult [thingUID=" + thingUID + ", properties=" + properties + ", flag=" + flag + ", label="
-                + label + ", bridgeUID=" + bridgeUID + "]";
+                + label + ", bridgeUID=" + bridgeUID + ", ttl=" + timeToLive + ", createdAt=" + createdAt + "]";
+    }
+    
+    @Override
+    public long getCreatedAt() {
```

Comment:
Originally I named it "timestamp". I changed it on Dennis' suggestion. I have the feeling, that I was not so wrong at all.... Should I rename it to timestamp?